### ISSUE TITLE
Mobile - "Manage" not fitting screen

### ISSUE THREAD

**Issue by dianaclem** [Chicago/opengrid#107]
Manage function doesn't fit on the screen for Samsung
![screenshot_2015-09-02-12-07-47 mobile manage screen](https://cloud.githubusercontent.com/assets/12613503/9638381/16f33cb0-516c-11e5-9266-322cb3971fec.png)
![screenshot_2015-09-02-12-10-54 sideways](https://cloud.githubusercontent.com/assets/12613503/9638390/220b2d1a-516c-11e5-99de-4f72155df6c5.png)

**Comment by tomschenkjr** [comment]
Is this still open?

**Comment by dianaclem** [comment]
Yes.  If screen is landscape, it renders all fields but if portrait it does not.   Was not a priority to address for 1.0

**Comment by tomschenkjr** [comment]
Thanks.
